src/java.desktop/share/classes/javax/print/PrintServiceLookup.java

```diff
@@ -344,32 +343,11 @@ private static ArrayList<PrintServiceLookup> getAllLookupServices() {
             } else {
                 listOfLookupServices = initListOfLookupServices();
             }
-            try {
-                java.security.AccessController.doPrivileged(
-                     new java.security.PrivilegedExceptionAction<Object>() {
-                        public Object run() {
-                            Iterator<PrintServiceLookup> iterator =
-                                ServiceLoader.load(PrintServiceLookup.class).
-                                iterator();
-                            ArrayList<PrintServiceLookup> los = getListOfLookupServices();
-                            while (iterator.hasNext()) {
-                                try {
-                                    los.add(iterator.next());
-                                }  catch (ServiceConfigurationError err) {
-                                    /* In the applet case, we continue */
-                                    if (System.getSecurityManager() != null) {
-                                        err.printStackTrace();
-                                    } else {
-                                        throw err;
-                                    }
-                                }
-                            }
-                            return null;
-                        }
-                });
-            } catch (java.security.PrivilegedActionException e) {
+            Iterator<PrintServiceLookup> iterator = ServiceLoader.load(PrintServiceLookup.class).iterator();
+            ArrayList<PrintServiceLookup> los = getListOfLookupServices();
+            while (iterator.hasNext()) {
+                los.add(iterator.next());
```

Comment:
Any reason for removing the try-catch block which is handling `ServiceConfigurationError` exception?